docs/man/encrypt/_index.md

```diff
@@ -27,7 +27,7 @@ command:
       default: /kas
     - name: with-assertions
       description: >
-        EXPERIMENTAL: JSON string of assertions to bind metadata to the TDF. See examples for more information.
+        EXPERIMENTAL: JSON string or path to JSON file of assertions to bind metadata to the TDF. See examples for more information.
```

Comment:
I think we should only support the file method.  A user may not be aware of the risks of specifying the key as a command argument.

We could consider adding argument support after #417 is implemented.

Reply:
i think keeping the json string options was just to maintain backward compatability, we can add some more documentation here to emphasize that its deprecated and the file path is the more safe options